src/controller/controller.c

```diff
@@ -1051,6 +1245,7 @@ static int controller_property_get_log_target(
 static const sd_bus_vtable controller_vtable[] = {
         SD_BUS_VTABLE_START(0),
         SD_BUS_METHOD("ListUnits", "", NODE_AND_UNIT_INFO_STRUCT_ARRAY_TYPESTRING, controller_method_list_units, 0),
+        SD_BUS_METHOD("ListUnitFiles", "", "a(sss)", controller_method_list_unit_files, 0),
```

Comment:
Please create macros similar to the ones for `ListUnits` (e.g. `NODE_AND_UNIT_INFO_STRUCT_ARRAY_TYPESTRING` for the array definition) in [protocol.h](https://github.com/eclipse-bluechi/bluechi/blob/main/src/libbluechi/common/protocol.h). 
(The same also for other occurences like in agent.c etc.)